README.md

```diff
@@ -30,6 +31,19 @@ $ rvm install ruby-2.4.2
 $ rvm use ruby-2.4.2
 ```
 
+**Java and jEnv**
+
+- Download and install [JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html)
```

Comment:
This page contains the paragraph:

> WARNING: These older versions of the JRE and JDK are provided to help developers debug issues in older systems. They are not updated with the latest security patches and are not recommended for use in production.

Does Community work with OpenJDK? If so, maybe this should link to https://adoptopenjdk.net/installation.html .